BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -121,205 +121,15 @@ smartConfirm = (logo, titleColorized, text, functions) ->
-    )
-
-#COLORPICKER ENABLED
-activateColorPicker = (element) ->
-  element.minicolors()
-
-#HELPER ENABLED
-activateHelper = (element) ->
-  title = element.attr('data-original-title').split('\\n').join('\n')
-  element.attr 'data-original-title', title
-  element.tooltip()
-
-#ACTIVATE HTML5 VALIDATION FOR HIDDEN
-activateHidden = (hidden) ->
-  hidden.addClass('focusable').attr('type', 'text')
-
-#ACTIVATE DATEPICKER
-activateDatepicker = (elements) ->
-  if $.fn.datepicker
-    elements.each ->
-      $this = $(this)
-
-      dataDateFormat = $this.data('dateformat') or 'yyyy-mm-dd'
-      convertFormatDay =
-        'EEEE' : 'DD'
-        'EE' : 'D'
-        'E' : 'D'
-        'D' : 'o'
-      convertFormatMonth =
-        'MMMM' : 'MM'
-        'MMM' : 'M'
-        'MM' : 'mm'
-        'M' : 'm'
-      convertFormatYear =
-        'Y' : 'yy'
-        'yyyy': 'yy'
-        'y' : 'yy'
-
-      convertFormat = (formats, dateFormat) ->
-        for format of formats
-            dateReplace = dateFormat.replace(new RegExp(format, 'g'), formats[format]);
-            return dateReplace if dateReplace != dateFormat
-        return dateFormat
-
-      dataDateFormat = convertFormat(convertFormatYear,dataDateFormat);
-      dataDateFormat = convertFormat(convertFormatMonth,dataDateFormat);
-      dataDateFormat = convertFormat(convertFormatDay,dataDateFormat);
-
-      $this.datepicker
-        dateFormat: dataDateFormat
-        prevText: '<i class="fa fa-chevron-left"></i>'
-        nextText: '<i class="fa fa-chevron-right"></i>'
-      $this = null
-      return
-
-#ACTIVATE CONTENT FILTER
-ajaxRefreshForm = (elements, view, form) ->
-  elements.on 'click', ->
-      $('form', view.$el).replaceWith('<h1 class="spin"><i class=\"fa fa-cog fa-spin\"></i> Loading...</h1>')
-      form.ajaxSubmit
-        method: 'POST'
-        context: view
-        success: (response) ->
-          $('.spin', @$el).replaceWith(response)
-          activateForm(@, $('form', @$el))
-    return
-
 #ACTIVATE FORM JS
 activateForm = (view, form) ->
-  activateSelect2(elements) if (elements = $(".select2", form)) && elements.length > 0
-  activateToken(elements) if (elements = $(".select-boolean", form)) && elements.length > 0
-  activateOrchestraNodeChoice(elements) if (elements = $(".orchestra-node-choice", form)) && elements.length > 0
-  activateColorPicker(elements) if (elements = $(".colorpicker", view.el)) && elements.length > 0
-  activateHelper(elements) if (elements = $(".helper-block", form)) && elements.length > 0
-  activateTinyMce(view, elements) if (elements = $("textarea.tinymce", form)) && elements.length > 0
-  activateHidden(elements) if (elements = $("input[type='hidden'][required='required']", form)) && elements.length > 0
-  activateDatepicker(elements) if (elements = $(".datepicker", form)) && elements.length > 0
-  ajaxRefreshForm(elements, view, form) if (elements = $(".refresh-form", form)) && elements.length > 0
+  try
```

Comment:
I agree with @itkg-alavieille : with that try catch, a little error somewhere freeze the entire process.

Plus it don't give any clue to the dev of what is causing the problem